capability/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright 2013 Suryandaru Triandana <syndtr@gmail.com>
```

Comment:
We can add a line above this "Copyright the authors" (which would apply to new contributions)

Reply:
Done; PTAL.